CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Introduction
+
+### Welcome dear contributor!
+
+First off, thank you for considering contributing to xcodeproj. It's people like you that make xcodeproj such a great tool.
+
+### Why reading the guidelines is important
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### A good contribution
+
+xcodeproj is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into xcodeproj itself.
+
+### A bad contribution
+
+Please, don't use the issue tracker for support questions. For that purpose use Stack Overflow tagging your threads with `#xcodeproj`.
+
+# Ground Rules
+
+* Ensure that changes in the project are tested.
+* Before merging, all the checks on GitHub should pass.
```

Comment:
What did you mean?

Reply:
Travis-CI mostly. Would you rephrase it differently @artemnovichkov?